nixos/modules/services/networking/clightning.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.clightning;
+
+  mkLightningNode = cfg: network:
+    let
+      configFile = pkgs.writeText "clightning-${network}.conf" cfg.config;
+    in
+    {
+      description = "clightning ${network} node";
+
+      # clightning requires bitcoin-cli to talk to either spruned or bitcoind
+      path = [ pkgs.altcoins.bitcoind ];
+
+      after = [ "network.target" ];
+      wantedBy = optional cfg.autoStart "multi-user.target";
+
+      serviceConfig.ExecStart = "${pkgs.clightning}/bin/lightningd --lightning-dir=${cfg.dataDir} --conf=${configFile}";
+      serviceConfig.Restart = "on-abort";
+    };
+in
+{
+  options = {
+    services.clightning = {
+      networks = mkOption {
```

Comment:
The name `networks` seems to be a leftover from openvpn. I think `nodes` would be more fitting.